tests/functional_tests/t0_main/langchain/t1_v1.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+import sys
+
+import wandb
+
+
+def simple_fake_test():
+    from langchain.chains import LLMChain
+    from langchain.llms.fake import FakeListLLM
+    from langchain.prompts import PromptTemplate
+
+    llm = FakeListLLM(responses=[f"Fake response: {i}" for i in range(100)])
+
+    prompt = PromptTemplate(
+        input_variables=["product"],
+        template="What is a good name for a company that makes {product}?",
+    )
+
+    chain = LLMChain(llm=llm, prompt=prompt)
+
+    for i in range(10):
+        chain(f"q: {i} - {datetime.datetime.now().timestamp()}")
+
+
+def main():
+    if sys.version_info < (3, 8):
```

Comment:
i don't think you are testing the WandbTracer integration, since we run base function tests only on py=3.7 see tox.ini

Reply:
ARRRG

Reply:
do you want to create a dedicate shard for your tests? another option is to add 3.8 to our base testings

Reply:
forgot to mention you need to update the number of builds [here](https://github.com/wandb/wandb/blob/main/.codecov.yml#L5-L14)